<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProduct</class>
 <widget class="QDialog" name="AddProduct">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Назва:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Код:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Каталог:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_16">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ТНВЕД:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Одиниці:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLineEdit" name="line_name">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_code">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_catalog">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_tnved">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_unit">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="btn_add">
       <property name="font">
        <font>
         <family>Alef</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Додати</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_reject_3">
       <property name="font">
        <font>
         <family>Alef</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Назад</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_reject_3</sender>
   <signal>clicked()</signal>
   <receiver>AddProduct</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
